teuthology/nuke/__init__.py

```diff
@@ -245,11 +245,11 @@ def nuke(ctx, should_unlock, sync_clocks=True, noipmi=False, keep_logs=False, sh
     with parallel() as p:
         for target, hostkey in ctx.config['targets'].items():
             status = get_status(target)
-            if ctx.name and ctx.name not in status.get('description', ""):
+            if ctx.name and ctx.name not in (status.get('description') or ""):
                 total_unnuked[target] = hostkey
                 log.info(
                     f"Not nuking {target} because description doesn't match: "
-                    f"{ctx.name} != {status['description']}"
```

Comment:
This seems like the actual fix for the exception

Reply:
If you visit the Sentry event I linked in the commit message you can see that the error is on line 248